mk/spksrc.spk.mk

```diff
@@ -84,7 +84,6 @@ $(WORK_DIR)/INFO:
 	$(create_target_dir)
 	@$(MSG) "Creating INFO file for $(SPK_NAME)"
 	@echo package=\"$(SPK_NAME)\" > $@
-	@echo thirdparty=\"yes\" >> $@
```

Comment:
I'm just wondering do we support older DSM versions, like really old 3.x or even 4.x where this could still be used? 
If not I'm very happy removing it completely 

Thanks for the quick updates @th0ma7 👌

Reply:
I don't believe we will ever be releasing any DSM 3.x or 4.x spk ever.  That's why I removed it entirely, further more as we lost all old packages during the web server migration.  Presumably similar hooks will disappear for DSM5 once DSM7 comes in.